### PR TITLE
Update MSS in DG

### PR DESCRIPTION
**Pull request by siriousguy**
#Fixes 243

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -598,7 +598,36 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-**Use case: UC11 - Clearing Address or Wedding Book**
+**Use case: UC11 - Viewing a wedding**
```
Comment (ywllowsensor) [inline docs/DeveloperGuide.md:601]: Perhaps this is the same is UC05?

Comment (ywllowsensor) [reply docs/DeveloperGuide.md:601]: Maybe you can change the name to "Viewing a Wedding" to reduce ambiguity of what the use case represents

**Review by ywllowsensor — COMMENTED** [review]
LGTM overall, just a minor nit

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (ywllowsensor) [reply docs/DeveloperGuide.md:601]: The name of UC05*

**Comment by siriousguy** [comment]
Similar to previous UC

**Pull request closed without merging**